### PR TITLE
WIP [SAMSON-218] adding stage name deploy info screens

### PR DESCRIPTION
**Pull request by irwaters**
* Add description here
adding the stage name to the various info screens about deployments.  this does exist as part of the summary, but there is want for it to be there specifically.  

* Add any screenshots if you are touching the UI
screen shot 2016-03-30 at 3 23 09 pm

* Remember to add unit tests

/cc @zendesk/samson

### Tasks
 - [ ] :+1: from team

### References
 - Jira link:
https://zendesk.atlassian.net/browse/SAMSON-218

### Risks
- Level: Low/Med/High
Low

### CODE REVIEW

File: app/views/projects/_deploy.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
       <%= content_tag :span, "Hotfix!", class: "label label-hotfix" if deploy.changeset.hotfix? %>
       <%= link_to "#{deploy.summary}", [@project || deploy.project, deploy] %>
     </td>
+    <td><%= link_to deploy.stage.name, [deploy.project, deploy.stage] %></td>
```
Comment (grosser) [inline app/views/projects/_deploy.html.erb]: should not show when viewing a stage page -> `if @stage` might do the trick

**Comment by irwaters** [comment]
bump

File: app/views/projects/_deploy.html.erb [hunk 2]
```diff
@@ -8,6 +8,9 @@
       <%= content_tag :span, "Hotfix!", class: "label label-hotfix" if deploy.changeset.hotfix? %>
       <%= link_to "#{deploy.summary}", [@project || deploy.project, deploy] %>
     </td>
+    <% unless @stage %>
+     <td><%= link_to deploy.stage.name, [deploy.project, deploy.stage] %></td>
```
Comment (grosser) [inline app/views/projects/_deploy.html.erb]: 2-space indent plz

**Comment by grosser** [comment]
:+1:

**Pull request merged**